### ISSUE TITLE
[BUG] Langauge choice window modal and unclosable

### ISSUE THREAD

**Issue by jfalconnet** [PCGen/pcgen#6517]
**Describe the bug**
The window to select language is stick. When I click Cancel or Ok nothing happends, and I cant do anything else I have to kill the app.

**To Reproduce**
1. Lauch PCGEN (6.09.05)
2. Click on Sources > Load Sources SRD 3.5 for Players
3. Click on New
4. Set Intelligence to 12
5. On the left of summary page, click the "Add Bonus Language"
6. Click ok. (or cancel)
7. The Add Bonus Language does not close, and nothing respond anymore.

The behavior is the same if we have chosen a Race before (and thus have chosen a possible language)
The behavior is the same through Add Language via Skill point
The behavior is the same through "Feats & Abilities" > Misc > GM Awards > Add Language : but only if we choose "cancel" (with other sources that allow this GM awards)

**Expected behavior**
The window should close and we should be able to go on.

**Screenshots**
![PcgenLangBug](https://user-images.githubusercontent.com/5015181/120888658-c7876580-c5f9-11eb-8b20-f2fe3b4b67a6.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 
 - Version: 20.04.2 LTS

**Additional context**
For the "Feats & Abilities" > Misc > GM Awards > Add Language"  way, there is a misspell : the window title is "Langauge choice" instead of "Language choice".

**Comment by pokemonpasta** [comment]
Reproducible in 6.09.05, Ubuntu 20.04, openjdk-11. I should let you know though that 6.09.05 does have quite a few issues with stuff like pop-up windows hanging indefinitely, due to undergoing work to transition the UI from Java Swing to JavaFX. Development is slow across the board in PCGen, so if you like, 6.08.00RC8 is also supported and doesn't have a lot of these issues.

**Comment by grimreaper** [comment]
Not reproduced on `master`. In my screen I see this:
<img width="788" alt="image" src="https://user-images.githubusercontent.com/433817/134437882-10b892ad-be2f-497c-abf4-0a3f4ffb9dee.png">

but am able to hit "okay" or "close"

**Comment by grimreaper** [comment]
The referenced bug was a _partial_ fix

**Comment by Vest** [comment]
@jfalconnet you are correct regarding the empty window - the bonus languages become available, when you select a race **and** add an age. The application recalculates something.
I can investigate this, but there is a workaround :)
